using-mobile-extensions/adobe-audience-manager/audience-manager-api-reference.md

```diff
@@ -335,7 +418,52 @@ Audience.signalWithData(traits, visitorProfileCallback);
 ```
 {% endtab %}
 
-{% tab title="iOS" %}
+{% tab title="iOS (AEP 3.x)" %}
+
+### signalWithData
+
+This API sends Audience Manager a signal with traits and returns the matching segments for the visitor in a callback.
+
+Audience Manager sends the AAM UUID in response in initial signal call. The AAM UUID is persisted in `NSUserDefaults` and is sent by the SDK in all subsequent signal requests. If available, the Experience Cloud ID \(MID\) is also sent in each signal request with the DPID and the DPUUID. The visitor profile that Audience Manager returns is saved in `NSUserDefaults`ß and is updated with every signal call.
```

Comment:
We don't need to escape the parentheses. So we can just make it "... the Experience Cloud ID (MID) is also sent..."

Reply:
Also, there's a random ß that has made its way into the sentence.

Reply:
Updated.